### ISSUE TITLE
Front Page Makeover - Part 1

### ISSUE THREAD

**Issue by TCEmkenn** [Third-Coast/website#23]
Roughly in order from "Top" of the page to "Bottom" of the page, in sections:

**Audio Player**

- [x] Can this be hidden until you click on something to listen to it, and then it appears? Maybe an arrow or something indicates that it's hidden there? (Does this make sense?)

**Nav Bar**

- [x] Get rid of green line above nav bar

- [x] Make the nav bar sticky

- [x] Get rid of subtle line markings around nav bar

- [x] Can you remove "News & Events" from the nav bar 

- [x] Can you remove "Playlists" from the nav bar (see other issue!)

**Header Photo**

- [ ] Can the space that this header image occupies change at all?

**Listen**

Okay, this is going to get kind of combined with the row beneath it (see below), and these individual pieces are going to be the baseline idea for "cards"?

But, to replace this SECTION, we want to make it possible to put some kind of big statement about Third Coast in 2020 — kind of an about, probably 2-4 sentences of text here. Not sure what the header is yet (maybe no header!). Is that something you could mock up?

**News and Events**

- [ ] Can you get rid of the "News and Events" section and replace it with what we've been talking about — a place where different "types" of things can exist side by side in "cards" that look similar to the "Listen" cards", but have "standard" information for each? (Will go into more detail on this below)!

- [ ] Can a header live above this that says "What's going on at Third Coast"

- [x] Can this section have a horizontal scroll, instead of being stacked vertically? 

**Foundation Support**

- [x] Can you flip the location of this so it goes above the footer? (TBH, I want this to look a little more integrated, but we're trying to figure out how to do that? IF you have any ideas, open for feedback)

**Footer**

- [x] Get rid of phone number & address from footer

- [x] Can you move ShortDocs to the footer, and put it under Producer Index?

- [ ] Is there a way to make a sign-up box for our newsletter embedded into the website directly? (We use myemma.com as our newsletter service.) So instead of clicking on "Third Coast Newsletters", it could be "Sign Up For Producer News" and there could be a First Name Box, Last Name Box, and Email Box?

- [x] Can you remove the word "Connect" above the social media icons?

- [ ] Can you re-order the footer so that it goes, top to bottom: 

- Producer News Sign-Up Box (Horizontal)

Then, in a row beneath that, left to right: 

-  Third Coast International Audio Festival
-  List of programs (I may come back to edit this to make more distinct lists): 
- Support 

Then, in a row underneath that: 

- social media icons, in a horizontal row

Then, in a row underneath that: 

- (c) 2020 Third Coast International Audio Festival Terms and Conditions | Privacy Policy

**Comment by brendanmetzger** [comment]
Couple notes to peruse..


Re Audio player: I checked the layout of the template to see if the panel can open/close, but the empty spot the player occupies is the page's margin, and changing the margin will affect the wrapping of all the text to the left, which us usually disorienting. That said, am I getting this wrong? There may be a bug I haven't come across yet, but the player should be hidden until you click a play button somewhere else on the page.

Re. Header photo: I'm guessing that could change a little, but all banners in place now were supplied by the designer, so they somewhate fixed in size—if they were text or dynamically composed in some other way then they could change size to whatever necessary.

Re email signup: I looked in the docs, and I think [Emma has a modal option](https://help.myemma.com/s/article/Customize-your-lightbox-form) which will pop up when you click instead of taking you to another page. At any rate, it looks like the sign up process is somewhat dictated by their offerings.

**Comment by brendanmetzger** [comment]
@TCEmkenn — Emma does have an API that allows signups, FYI, so the newsletter thing is possible, but would have to be developed.

**Comment by TCEmkenn** [comment]
I just added some questions about "cards." How far have you gotten with that idea?

**Comment by TCEmkenn** [comment]
It looks like when the dev site when live, some of the elements that you changed (above) stayed the same:

**Nav Bar**

[ ] Get rid of green line above nav bar

[ ] Make the nav bar sticky

[ ] Get rid of subtle line markings around nav bar

**Can you push those live?**

**Comment by brendanmetzger** [comment]
Those are all controlled by the stylesheet, which depending on your browser, get's cached pretty hard b/c it rarely changes. If you are using chrome, try command-shift-R to do a "hard refresh" and I think those changes will show up.

**Comment by TCEmkenn** [comment]
Omg, you're a genius! That worked.

Wow. OK. That's so annoying that it gets cached pretty hard! How do we make
sure it isn't cached for our audience? (Should we tell them in a newsletter
to do that???)

*Emily S. Kennedy* | Program Director | Third Coast Festival
<http://thirdcoastfestival.org/>

ekennedy@thirdcoastfestival.org | Pronouns: She/Her/Hers

**Comment by brendanmetzger** [comment]
Yeah, its super annoying, and I'll spare the digression into why that's the case, but 99% of users wouldn't be affected.

I'll manually force the cache to reset when most of the changes are in place, but that's somewhat tedious to do for every push... I do have recent applications that do this automatically when a change is detected, but that's a 3.0 level stuff unfortunately.

**Comment by TCEmkenn** [comment]
GOT IT. Ok. That makes a ton of sense!!! More soon.


*Emily S. Kennedy* | Program Director | Third Coast Festival
<http://thirdcoastfestival.org/>

ekennedy@thirdcoastfestival.org | Pronouns: She/Her/Hers

**Comment by TCEmkenn** [comment]
For now, can you switch the "New & Events" & "Listen" section on the homepage of www.thirdcoastfestival.org! Thank you.

**Comment by brendanmetzger** [comment]
added back in the footer stuff.

As for the news & events, listen section, are you referring to the card collections? I'm still theorizing how to make that happen but I'm trying to knock down as many other tasks as I can first, as I think it will require most of my focus for a minute when I actually get going.

**Comment by TCEmkenn** [comment]
Thank you!

I think, because we're doing these announcements tomorrow, we'd love to see
what it would look like if Listen & News are just flipped for now on the
homepage—nothing fancier than that? Agreed that the card collections seem
like a much bigger project!

*Emily S. Kennedy* | Program Director | Third Coast Festival
<http://thirdcoastfestival.org/>

ekennedy@thirdcoastfestival.org | Pronouns: She/Her/Hers

**Comment by brendanmetzger** [comment]
phew—ok, that's easy enough.

I'm simultaneously trying to get the page to load faster, and have switched the typeface load requirements to be less intense. Anyway, in the screenshot, new is on the left. I think the title face, while similar, is exceptionally beautiful in the details. I've spent some time trying to find looking for diversity in the type community (its largely european white dudes), but that one is called Dapifer by [Joshua Darden](https://en.wikipedia.org/wiki/Joshua_Darden). It's also beautiful italicized, as seen in 'Third Place' in the header. (still playing with this to make it stand out but).

![Screen Shot 2020-09-17 at 1 56 39 PM](https://user-images.githubusercontent.com/4213/93516971-65ae9080-f8f0-11ea-8d71-247b52616078.png)

**Comment by TCEmkenn** [comment]
Hey hey!

This is so, so cool—gonna look more over the weekend, and think on it!

Send me any resources you're checking out, if you want to? I'm geekily into
font stuff and I LOVE the idea of using a font made by someone who is not a
white European!! (I used to work for the guy who runs Type Network
<https://www.typenetwork.com/> (he ran a small magazine, too) and ... there
were a lot of verrrrrrry snarky designers around!!!)

EK

*Emily S. Kennedy* | Program Director | Third Coast Festival
<http://thirdcoastfestival.org/>

ekennedy@thirdcoastfestival.org | Pronouns: She/Her/Hers

**Comment by TCEmkenn** [comment]
OK, I need to look over some of your notes — I'm so sorry they slipped through the cracks! Just added a few more questions above!
